Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
It does. However, the upstream of both `execline` and `skalibs` don't support shared libraries on OS X (and explicitly don't want to, cfr their README and questions on the mailing list). That said, the static libs are built so they don't incur much waste of space, so the net result shouldn't be that different in size aspect. 

It could probably be patched to build and use shared libs on OS X, but it would never be integrated upstream, so it would need to stay maintained by homebrew, which isn't worth the effort in this case i think.